etl/steps/export/explorers/wash/latest/water_and_sanitation.config.yml

```diff
@@ -0,0 +1,2040 @@
+        Basic drinking water is water from an improved source, where the collection time is not more than 30 minutes for a roundtrip.
+  - dimensions:
+      resource: drinking_water
+      residence: total
+      level_of_use_access: limited
+      relative_to_population: share_of_population
+    indicators:
+      y:
+        - catalogPath: who#wat_lim__residence_total
+          display:
+            name: Usage of limited drinking water services
+            type: Numeric
+            unit: '%'
+            shortUnit: '%'
+            additionalInfo: |-
+              Limited drinking water is water from an improved source, where the collection time exceeds 30 minutes for a roundtrip.\nImproved drinking water sources are those that have the potential to deliver safe water. They include piped water, boreholes or tubewells, protected dug wells, protected springs, rainwater, and packaged or delivered water.
+            colorScaleScheme: YlOrBr
+            colorScaleNumericBins: 5,,;10,,;20,,;30,,;40
+            colorScaleEqualSizeBins: false
+            colorScaleNumericMinValue: 0
+    config:
+      title: Share of the population using limited drinking water services
+      subtitle: Limited drinking water is water from an improved source, where the collection time exceeds 30 minutes for
+        a roundtrip.
+  - dimensions:
+      resource: drinking_water
+      residence: total
+      level_of_use_access: unimproved
+      relative_to_population: share_of_population
+    indicators:
+      y:
+        - catalogPath: who#wat_unimp__residence_total
+          display:
+            name: Usage of unimproved drinking water sources
+            type: Numeric
+            unit: '%'
+            shortUnit: '%'
+            additionalInfo: An unimproved drinking water source is water from an unprotected dug well or unprotected spring.
+            colorScaleScheme: YlOrRd
+            colorScaleNumericBins: 5,,;10,,;20,,;30,,;40,,;50,,;60
+            colorScaleEqualSizeBins: false
+            colorScaleNumericMinValue: 0
+    config:
+      title: Share of the population using unimproved drinking water sources
+      subtitle: An unimproved drinking water source is water from an unprotected dug well or unprotected spring.
+  - dimensions:
+      resource: drinking_water
+      residence: total
+      level_of_use_access: no_facilities
+      relative_to_population: share_of_population
+    indicators:
+      y:
+        - catalogPath: who#wat_ns__residence_total
+          display:
+            name: No usage of drinking water facilities
+            type: Numeric
+            unit: '%'
+            shortUnit: '%'
+            additionalInfo: |-
+              Having no access to drinking water means people rely on surface water: directly from a river, dam, lake, pond, stream, canal or irrigation canal.
+            colorScaleScheme: YlOrRd
+            colorScaleNumericBins: 2.5,,;5,,;10,,;25,,;50
+            colorScaleEqualSizeBins: false
```

Comment:
I think this should be true as default (as it is on Grapher)
<img width="1305" alt="Screenshot 2025-04-15 at 16 27 59" src="https://github.com/user-attachments/assets/cb0c1be9-e4b9-46ce-ae62-91631984efd3" /> at the moment there are funky bins on staging

Reply:
Good catch! There were two columns with `colorScaleEqualSizeBins` in production which caused this. Fixed.